dev/conductor/core/lib/src/next.dart

```diff
@@ -142,14 +142,11 @@ class NextContext extends Context {
 
         await pushWorkingBranch(engine, state.engine);
       case pb.ReleasePhase.CODESIGN_ENGINE_BINARIES:
-        stdio.printStatus(<String>[
-          'You must validate pre-submit CI for your engine PR, merge it, and codesign',
-          'binaries before proceeding.\n',
```

Comment:
So is MPA when codesigning happens now?

Reply:
No, codesigning happens on engine postsubmit, and it's verified with a framework presubmit test.

Reply:
ahh, so the builders are like channel aware, and then flip the codesign bit when it knows it's a release?

Reply:
Yeah! Anything that is built from dart-internal's secure builders will be code signed, which our RC branches use.

Reply:
ahh, interesting. so master channel engine builds use the luci.flutter.prod pool, but RC branches use dart-internal?

Reply:
Yep! Longer term, luci.flutter.prod will converge with dart-internal to build all engine postsubmit artifacts. Then anything that could be used will be code signed.

Reply:
Very cool